README.md

```diff
@@ -18,6 +18,7 @@ export KUBECONFIG=/path/to/your/kube/config
 ```bash
 ./build.sh
 ```
+For more information about the build script ```./build -h```
```

Comment:
Better follow the structure in the document. For example use:
```bash
./build -h
```
instead of leaving it inline.

Reply:
Check the code above this line to see how to do it.